control-base/src/main/java/fi/nls/oskari/control/layer/GetWmsServicesHandler.java

```diff
@@ -127,7 +127,7 @@ private JSONObject makeMapLayersJson() {
 //                mapProperties.put("descriptionLink", ml.getDescriptionLink());
                 mapProperties.put("legendImage", ml.getLegendImage());
 
-                mapProperties.put("inspireTheme", ml.getInspireTheme().getId());
+                mapProperties.put("inspireTheme", ml.getMaplayerGroup().getId());
```

Comment:
`inspireTheme` is out of sync. Could temporarily add another key with more suiting name and then remove this line after frontend code is up to date with these changes.